public/app/plugins/panel/table/module.tsx

```diff
@@ -104,6 +104,70 @@ export const plugin = new PanelPlugin<Options, FieldConfig>(TablePanel)
           name: 'Hide in table',
           defaultValue: undefined,
           hideFromDefaults: true,
+        })
+        .addBooleanSwitch({
+          path: 'footer.show',
+          category: [footerCategory],
+          name: 'Show table footer',
+          defaultValue: defaultOptions.footer?.show,
+        })
+        .addCustomEditor({
+          id: 'footer.reducer',
+          category: [footerCategory],
+          path: 'footer.reducer',
+          name: 'Calculation',
+          description: 'Choose a reducer function / calculation',
+          editor: standardEditorsRegistry.get('stats-picker').editor,
+          override: standardEditorsRegistry.get('stats-picker').editor,
+          defaultValue: [ReducerID.sum],
+          process: identityOverrideProcessor,
+          showIf: (cfg: FieldConfig) => cfg.footer?.show,
+          shouldApply: () => true,
+        })
+        .addBooleanSwitch({
+          path: 'footer.countRows',
```

Comment:
count itself is a reducer already, isn't it?

https://github.com/grafana/grafana/blob/2ece94bc50822c52ce220125377964798f3108fe/packages/grafana-data/src/transformations/fieldReducer.ts#L19